daemon/container_daemon.go

```diff
@@ -8,4 +8,5 @@ type ContainerDaemon interface {
 	DeleteContainer(hostname string) (bool, error)
 	CreateContainerBootstrapScript(container pfmodel.Container) (bool, error)
 	ValidateAndBootstrapContainer(container pfmodel.Container) (bool, error)
+	MigrateContainer(container pfmodel.Container) (success bool, ipaddress string, err error)
```

Comment:
All the other is using `relocate` terminology, why do we use `migrate` here?

Reply:
Actually the lxd client use term `migrate` container, so we wrap the function with the same name.